AcceptanceTests/pom.xml

```diff
@@ -19,11 +19,11 @@
 		<outputPath>Test-Reports/${env}/${timestamp}</outputPath>
 		<testngXml>android.xml</testngXml>
 		<sauceUser></sauceUser>
-		<sauceKey></sauceKey>
+		<sauceKeys></sauceKeys>
 		<sauceBuildName></sauceBuildName>
 		<deviceOS>android</deviceOS>
-		<appPath>sauce-storage:edxDebug.apk</appPath>
-		<osVersion>4.4</osVersion>
+		<appPath>/Users/divakarpatil/Desktop/edx-prod-release-1.0.04-45.apk</appPath>
+		<osVersion>4.4.2</osVersion>
 		<deviceName>Android Emulator</deviceName>
 		<appPackage>org.edx.mobile</appPackage>
 		<emailId>divakar.patil@claricetechnologies.com</emailId>
```

Comment:
Also, I just noticed this line - which isn't part of the change, but probably your email shouldn't be hardcoded in?